### PR TITLE
0.13.2 deploy bug fix

### PR DESCRIPTION
**Pull request by mateoclarke**
This PR is basically the same as https://github.com/cityofaustin/atd-moped/pull/544 which is already in prod.

It removes external kickstand dependency for GQL Abstract.

## Testing
**URL to test:** https://deploy-preview-542--atd-moped-main.netlify.app/moped/session/signin

**Steps to test:**

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#gid=776973707)

### CODE REVIEW

File: moped-editor/package.json [hunk 1]
```diff
@@ -105,5 +105,8 @@
     "eslint-plugin-react": "^7.20.3",
     "eslint-plugin-react-hooks": "^2.5.1",
     "prettier": "^1.19.1"
+  },
+  "engines": {
+    "npm": ">=6.0.0"
   }
-}
+}
```
Comment (frankhereford) [inline moped-editor/package.json:112]: Do you mind if I push up a commit to this branch to add this new line here?

**Review by frankhereford — APPROVED** [review]
Looks great - Thank you for compiling these!

bike-moped

**Review by chiaberry — APPROVED** [review]

File: moped-editor/package.json (hunk 1 above)
Comment (mateoclarke) [reply moped-editor/package.json:112]: sure, please go for it.

Comment (mateoclarke) [reply moped-editor/package.json:112]: nvm, let me do it quick, i already have the branch open and I want to go ahead and merge

**Pull request merged**